parts/linux/cloud-init/artifacts/aks-check-network.sh

```diff
@@ -100,7 +101,7 @@ function check_and_curl {
         if [ $response -ge 200 ] && [ $response -lt 400 ]; then
             logs_to_events "AKS.testingTraffic.success" "echo '$(date) - SUCCESS: Successfully tested $url with returned status code $response'"
             break
-        elif [ $response -eq 400 ] && ([ $url == "acs-mirror.azureedge.net" ] || [ $url == "eastus.data.mcr.microsoft.com" ]); then
+        elif [ $response -eq 400 ] && ([ $url == "eastus.data.mcr.microsoft.com" ]); then
```

Comment:
Are we still using $url here?

Reply:
Yeah, for one of those endpoints "400" is the expected code